pkgs/tools/misc/env-lscolors/default.nix

```
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation {
+  pname = "env-lscolors";
+  version = "unstable-2022-05-05"; # upstream don't do releases
+
+  src = fetchFromGitHub {
+    owner = "trapd00r";
+    repo = "LS_COLORS";
+    rev = "7271a7a8135c1e8a82584bfc9a8ebc227c5c6b2b";
+    sha256 = "0gs2qmdxvvgs5ck2j8b6i8dqc5q91m8xrvc2ajvlhcr7in0n9iw5";
+  };
+
+  # This is unique situation, so it is not really clear how to format and
+  # where to put result.
+  #
+  # Dircolors can output scripts source-able by different shells, but
+  # including them all is redundant and wastes output size; blessing one
+  # particular shell will make users of rest unhappy.
+  #
+  # Here is compromise: value of the variable is put into file, and user is
+  # expected to do something like LS_COLORS=$(cat ...). This is also
+  # daemontools-friendly.
+  installPhase = ''
+    eval "$(dircolors LS_COLORS)"
+    mkdir -p $out/share/env-lscolors/env $out/etc/profile.d
+    echo "$LS_COLORS" > $out/share/env-lscolors/env/LS_COLORS
+    cat << EOF > $out/etc/profile.d/env-lscolors.sh
+    lscolors_init() {
+      export LS_COLORS=\`cat $out/share/env-lscolors/env/LS_COLORS\`
+    }
+    EOF
```

```suggestion
    mkdir -p $out/share/lscolors
    cp LS_COLORS $out/share/lscolors/LS_COLORS
    dircolors -b LS_COLORS > $out/share/lscolors/lscolors.sh
```
we only really need this.